public/js/calendarPage.js

```diff
@@ -1,5 +1,13 @@
 const calendarEl = document.getElementById("calendar");
 const calendar = new FullCalendar.Calendar(calendarEl, {
+  eventClick: function(info) {
+    var eventObj = info.event;
+    $('#eventName').html(eventObj.title);
+    $('#eventLocation').html(eventObj.extendedProps.location)
+    $('#eventStart').html(eventObj.start)
+    $('#eventEnd').html(eventObj.end)
+    $('#calendarModal').modal();
```

Comment:
I'm pretty sure you could have written this without jQuery, doing something like this for each item:

`document.querySelector('#eventName').textContent = eventObj.title`